### DISCUSSION TITLE
OpenInv 4.3.1

### DISCUSSION THREAD

**Discussion by Jikoo** [Jikoo/OpenInv#150] [General]
## Supported Minecraft versions
1.19.4, 1.20

## What's Changed
* Update to Minecraft 1.20
* Drop support for Minecraft versions below 1.19.4
  * 1.19.4 is currently supported to make it easier for server admins to update gradually
* Fix loading users in deleted worlds on Paper (for real this time)
* Fix double chests not being deactivated properly
* Code health
* Fix/improve Actions

**Full Changelog**: https://github.com/Jikoo/OpenInv/compare/4.3.0...4.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Jikoo/OpenInv/releases/tag/4.3.1'>OpenInv 4.3.1</a>.</em>

**Comment by Themongoles78cbejnev** [comment]
que faut-il que je fasse pour pouvoir mettre se mod sur mon serveur aternos car quand je le met il ne veut pas et me redirige vers se site

**Reply by Jikoo** [reply]
Not sure exactly what you're trying to say, the translation is coming out a bit garbled and I don't speak French. Please use precise language and include more detail.

The impression I'm getting is that you're having trouble adding OpenInv to your server? Make sure you have downloaded the correct version for your server version. If you're getting directed to the 4.3.1 release, you are likely running 1.20(.1) and have installed an older version of OpenInv.

**Comment by Narutosamaa** [comment]
When will be openinv available for 1.20.1??

**Reply by Jikoo** [reply]
1.20.1 was a minor patch that contained almost no changes. OpenInv should already support it.
